coredns/README.md

```diff
@@ -70,6 +70,18 @@ EOF
 
 ```
 
+Running the operator locally:
+
+```bash
+export GOPATH=<go-path>
```

Comment:
Thanks for adding this @SomtochiAma !

We're using Go modules for this project so a user shouldn't need to set GOPATH. Do these instructions work with setting `GO111MODULE=on` instead of GOPATH=..? 

Instead of go build and running bin/manager, i'd suggest putting `make run` after `make install`. This ensures all files are generated.

Reply:
Okay. I will try to do it without setting GOPATH and if it works, I will update the PR.

Reply:
The "generate" command in the make file checks if the GOPATH is set else it exits with an error. Therefore make run need the GOPATH Set.

Reply:
Ohh- thank you! Let's go with this as it is then. We're still in the weird world where go modules exists, but not everywhere :)